scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -462,12 +462,14 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                 }
             }
 
+            let minTansaUrgency = $rootScope.config.tansa && $rootScope.config.tansa.urgency || 5;
```

Comment:
I will keep it close to that usage, imo those whole ansa specific part should be eventually moved away